integration-cli/docker_cli_export_import_test.go

```diff
@@ -18,12 +18,6 @@ func (s *DockerSuite) TestExportContainerAndImportImage(c *check.C) {
 		c.Fatal("failed to create a container", out, err)
 	}
 
-	inspectCmd := exec.Command(dockerBinary, "inspect", containerID)
```

Comment:
The only worry I have with removing this 'inspect' is that things might get racey - this 'inspect' call may have given the previous docker call time to complete.  I would suggest that, in addition to removing this 'inspect' call, you also remove the `-d` on the `docker run` to ensure that by the time we get here the container is not only created but done running. IMO we should use `--name ...` more often (w/o `-d`) and use that name instead of the id.  Its easier in cases where we don't really need the container to be detached.